cmd/bundle.go

```diff
@@ -53,7 +53,13 @@ var bundleTemplateRefreshCmd = &cobra.Command{
 var bundleNewCmd = &cobra.Command{
 	Use:   "new",
 	Short: "Create a new bundle from a template",
-	RunE:  runBundleNew,
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
this will be rough in a stack trace. We should keep the named function and put this logic within the runBundleNew command.